.github/workflows/artifacts_CI.yml

```diff
@@ -0,0 +1,119 @@
+name: Artifacts CI
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    tags: ["*"]
+    paths:
+      - "ray_julia_jll/Artifacts.toml"
+      - ".github/workflows/artifacts_CI.yml"
+  pull_request:
+    paths:
+      - "ray_julia_jll/Artifacts.toml"
+      - ".github/workflows/artifacts_CI.yml"
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on:
+      labels: ubuntu-20.04-16core
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - "1.8"  # Version used with MVP
+          - "1.9"  # Latest release
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+      env:
+        ray_julia_jll_dir: ./ray_julia_jll
+        ray_dir: ./ray_julia_jll/deps/ray
+      steps:
+      - uses: actions/checkout@v3
+      - name: Clone Ray repository
```

Comment:
You've got a syntax issue: https://github.com/beacon-biosignals/Ray.jl/actions/runs/6317214253. YAML normally doesn't care if this is indented here but maybe GHA does? In any case I would indent everything under `steps:`

Reply:
ah ok I'll fix this then ping you again

Reply:
seems to be working now!